### PR TITLE
Updating the protobufs

### PR DESCRIPTION
**Pull request by tinystarinagalaxy**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+src/
```
Comment (tinystarinagalaxy) [inline .gitignore]: im not sure about this

File: bxserum/proto/__init__.py [hunk 2]
```diff
@@ -1 +0,0 @@
-from .api import *
```
Comment (aspin) [inline bxserum/proto/__init__.py:1]: revert this change. I couldn't figure out make this happen automatically

File: .gitignore (hunk 1 above)
Comment (aspin) [reply .gitignore]: what's causing this? shouldn't be needed

File: bxserum/proto/__init__.py (hunk 2 above)
Comment (tinystarinagalaxy) [reply bxserum/proto/__init__.py:1]: this happens after i run `./generate_proto.sh`

Comment (tinystarinagalaxy) [reply bxserum/proto/__init__.py:1]: i put it back.

File: .gitignore (hunk 1 above)
Comment (tinystarinagalaxy) [reply .gitignore]: fixed

**Review by aspin — APPROVED** [review]
